app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,45 @@
+class Api::V1::UsersController < ApplicationController
+  skip_before_action :authenticate_user, except: [:deactivate, :index, :tasks] 
+  before_action :require_admin, only: [:deactivate, :index]
+
+  def deactivate
+    email = params[:email]
```

Comment:
dont need this variable if we are just using it in one place, just directly do `deactivate_user(params[:email])`